_maps/RandomRuins/SpaceRuins/Immortus.dmm

```diff
@@ -0,0 +1,3406 @@
+/obj/structure/filingcabinet,
+/obj/structure/sign/warning/biohazard{
+	pixel_y = 31
+	},
+/turf/open/floor/plasteel/darkpurple/side{
+	dir = 1
+	},
+/area/ruin/powered/Immortus)
+"dM" = (
+/obj/structure/filingcabinet,
+/turf/open/floor/plasteel/darkpurple/side{
+	dir = 1
+	},
+/area/ruin/powered/Immortus)
+"dN" = (
+/turf/open/floor/plasteel/darkpurple/corner{
+	dir = 1
+	},
+/area/ruin/powered/Immortus)
+"dO" = (
+/obj/structure/closet/crate/secure/loot,
+/turf/open/floor/plasteel/darkpurple/side{
+	icon_state = "darkpurple";
+	dir = 10
+	},
+/area/ruin/powered/Immortus)
+"dP" = (
+/obj/structure/frame,
+/turf/open/floor/plasteel/darkpurple/side{
+	icon_state = "darkpurple";
+	dir = 6
+	},
+/area/ruin/powered/Immortus)
+"dQ" = (
+/mob/dead/observer{
+	name = "Herbert West"
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"dR" = (
+/obj/structure/table/reinforced,
+/obj/item/gun/magic/wand/resurrection{
+	desc = "This wand uses healing magics to heal and revive. This one has been subjected to rigorous experimentation and only had a single-use left.";
+	max_charges = 1;
+	name = "faded wand of healing"
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"dS" = (
+/obj/item/twohanded/required/kirbyplants{
+	icon_state = "plant-22"
+	},
+/obj/effect/decal/immortus{
+	alpha = 180;
+	color = "#90BCD7";
+	pixel_x = -28;
+	pixel_y = 4
+	},
+/turf/open/floor/plasteel/white,
+/area/ruin/powered/Immortus)
+"dT" = (
+/obj/machinery/gulag_teleporter/immortus,
+/turf/open/floor/plating,
+/area/ruin/powered/Immortus)
+"dU" = (
+/obj/machinery/gulag_teleporter/immortus/exit,
+/turf/open/floor/plasteel/white,
+/area/ruin/powered/Immortus)
+"dV" = (
+/turf/open/floor/plating,
+/area/ruin/powered/Immortus)
+"ek" = (
+/obj/effect/turf_decal/stripes/asteroid/line,
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"fv" = (
+/obj/effect/spawner/structure/window/shuttle,
+/obj/structure/trap/chill,
+/turf/closed/indestructible/vault,
+/area/ruin/powered/Immortus)
+"fw" = (
+/obj/machinery/door/password/voice{
+	password = "Miskatonic"
+	},
+/turf/open/floor/plasteel/white,
+/area/ruin/powered/Immortus)
+"gH" = (
+/obj/structure/table/reinforced,
+/obj/item/clockwork/slab,
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"iJ" = (
+/obj/structure/table/wood/bar,
+/obj/item/gun/syringe/rapidsyringe,
+/obj/item/reagent_containers/glass/bottle{
+	name = "bottle of romerol";
+	pixel_x = 6;
+	pixel_y = -3
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"kd" = (
+/obj/machinery/iv_drip,
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"kw" = (
+/obj/item/ammo_casing/a357{
+	pixel_x = 5;
+	pixel_y = -9
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"oU" = (
+/obj/machinery/manned_turret/ultimate{
+	name = "Immortus containment turret"
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"rB" = (
+/obj/item/card/id/syndicate/nuke_leader{
```

Comment:
hmmm

Reply:
Intended, unless there's a better ID-class for this sort of exclusive access.

Reply:
You could probably use ACCESS_AWAY_MED (202) or something else that doesn't appear anywhere else.